doc/OCP_LOCK.md

```diff
@@ -0,0 +1,177 @@
+![LOCK Logo](https://chipsalliance.github.io/Caliptra/doc/images/logo/LOCK_logo_large.png)
+
+# OCP L.O.C.K.
```

Comment:
Wondering if we should have some sort of "versioning" added to this?  Something that speaks to the maturity of this prose.

Reply:
So far we've been treating these as somewhat living documents, and will periodically tag a release with a given version.